capstone/src/main/java/com/google/univiz/api/VisData.java

```diff
@@ -0,0 +1,15 @@
+package com.google.univiz.api;
+
+import java.util.List;
+
+/**
+ * This interface is responsible for all data management/processing specific for the visualization
```

Comment:
OOC, when you say 'visualization' are we talking about the charts? Cause mapping also could be considered a visualization. Maybe point out that it's the non-geographic visualizations. Or use other terms to describe it like chart or graph or tabular.